components/brave_wallet/browser/fil_response_parser_unittest.cc

```diff
@@ -20,11 +21,30 @@ TEST(FilResponseParserUnitTest, ParseFilGetBalance) {
   EXPECT_TRUE(brave_wallet::ParseFilGetBalance(json, &value));
   EXPECT_EQ(value, "10000000000000000000000000000");
 
-  json =
-      "{\"jsonrpc\":\"2.0\",\"id\":1,\"result\":"
-      "\"\"}";
+  json = "{\"jsonrpc\":\"2.0\",\"id\":1,\"result\":\"\"}";
   EXPECT_TRUE(brave_wallet::ParseFilGetBalance(json, &value));
   EXPECT_TRUE(value.empty());
 }
 
+TEST(FilResponseParserUnitTest, ParseFilGetTransactionCount) {
```

Comment:
let's test more possible number value in json, like negative integer, and also a non-integer number with fractions, to make sure they'll be false.

Reply:
done